runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationOverlay.java

```diff
@@ -80,6 +80,18 @@ public void renderItemOverlay(Graphics2D graphics, int itemId, WidgetItem itemWi
 					return;
 				}
 				break;
+			case ORE:
+				if (!config.showOres())
+				{
+					return;
+				}
+				break;
+			case GEM:
+				if (!config.showGems())
```

Comment:
all config calls in here should be cached to plugin (like it is in most overlays for us now)